quantity-pickers/src/main/java/com/decathlon/vitamin/compose/quantity/pickers/VitaminQuantityPickers.kt

```diff
@@ -69,59 +75,98 @@ object VitaminQuantityPickers {
+                        onClick = onAddClicked
+                    )
+                }
+                ErrorLabel(errorMessage = errorMessage)
+            }
+        }
+    }
+
+    @Composable
+    private fun ErrorLabel(
+        errorMessage: String?
+    ) {
+        errorMessage?.let {
+            Box(
+                modifier = Modifier
+                    .size(4.dp)
+            )
+            Row(
+                verticalAlignment = Alignment.CenterVertically
             ) {
-                IconButton(
-                    painter = painterResource(id = R.drawable.ic_vtmn_subtract_fill),
-                    contentDescription = stringResource(id = R.string.vtmn_subtract_button_description),
-                    enabled = subtractEnabled,
-                    colors = colors,
-                    shape = shape.copy(
-                        topEnd = CornerSize(0.dp),
-                        bottomEnd = CornerSize(0.dp)
-                    ),
-                    onClick = onSubtractClicked
+                Icon(
+                    modifier = Modifier
+                        .size(16.dp),
+                    painter = painterResource(id = R.drawable.ic_vtmn_information_line),
+                    contentDescription = stringResource(id = R.string.vtmn_information_icon_description),
+                    tint = VitaminTheme.colors.vtmnContentNegative
                 )
-
-                MiddleTextField(
+                Box(
                     modifier = Modifier
-                        .background(
-                            color = colors.backgroundColor.copy(
-                                alpha = if (!addEnabled && !subtractEnabled) VtmnStatesDisabled else VtmnStatesEnabled
-                            )
-                        )
-                        .fillMaxHeight()
-                        .weight(1.0f),
-                    addEnabled = addEnabled,
-                    subtractEnabled = subtractEnabled,
-                    colors = colors,
-                    textStyle = textStyle,
-                    value = value,
-                    onValueChange = onValueChange,
-                    editTextEnabled = editTextEnabled,
-                    keyboardOptions = keyboardOptions,
-                    keyboardActions = keyboardActions
+                        .size(8.dp)
                 )
-
-                IconButton(
-                    painter = painterResource(id = R.drawable.ic_vtmn_add_fill),
-                    contentDescription = stringResource(id = R.string.vtmn_add_button_description),
-                    enabled = addEnabled,
-                    colors = colors,
-                    shape = shape.copy(
-                        topStart = CornerSize(0.dp),
-                        bottomStart = CornerSize(0.dp)
-                    ),
-                    onClick = onAddClicked
+                Text(
+                    text = it,
+                    style = VitaminTheme.typography.text3,
```

Comment:
Please create a size class to save this typography, text field typography and icon sizes.